Content.Shared/Radio/Components/HeadsetComponent.cs

```diff
@@ -14,5 +14,5 @@ public sealed partial class HeadsetComponent : Component
     public bool IsEquipped = false;
 
     [DataField("requiredSlot")]
-    public SlotFlags RequiredSlot = SlotFlags.EARS;
+    public SlotFlags RequiredSlot = SlotFlags.EARS & SlotFlags.HEAD;
```

Comment:
The headset was removed from your borg PR, so you should be making no changes to HeadSetComponent